components/Molecule/graduates.html

```diff
@@ -0,0 +1,401 @@
+<style>
+html{
+	scroll-behavior: smooth;
+}
+.graduates_profile{
```

Comment:
It looks like this isn't part of a component. You can create a graduates component.